src/doublylinkedlist.c

```diff
@@ -1,6 +1,13 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+#ifdef __cplusplus
+#include <cstddef>
+#else
+#include <stddef.h>
+#include <stdbool.h>
+#endif
```

Comment:
this only ever gets compiled as "C". so #include <stdboo.h> is enough

Reply:
Removing in next iteration